grafana-formula/grafana/init.sls

```diff
@@ -2,7 +2,11 @@
 {%- set supported_vers = ['42.3', '12.3', '12.4', '12.5', '15.0', '15.1', '15.2', '15.3', '15.4', '15.5'] %}
 
 # check if supported
-{%- set supported = (grains['os_family'] == 'Suse' and grains['osrelease'] in supported_vers) %}
+{%- if (grains['os_family'] == 'Suse' and grains['osrelease'] in supported_vers) %}
+  {%- if not (salt['pkg.version']('patterns-uyuni_proxy') or salt['pkg.version']('patterns-suma_proxy') or salt['pkg.version']('patterns-suma_retail') or salt['pkg.version']('patterns-uyuni_retail')) %}
+    {%- set supported = True %}
+  {%- endif %}
+{%- endif %}
```

Comment:
Do you mean like that @aaannz ?

```suggestion
{%- if (grains['os_family'] == 'Suse' and grains['osrelease'] in supported_vers) %}
  {%- if not (salt['pkg.version']('patterns-uyuni_proxy') or salt['pkg.version']('patterns-suma_proxy') or salt['pkg.version']('patterns-suma_retail') or salt['pkg.version']('patterns-uyuni_retail')) %}
    {%- set supported = True %}
  {%- else %}
    os_not_supported:
      test.fail_without_changes:
        - name: "OS not supported!"
  {%- endif %}
{%- endif %}
```

Reply:
Yes. Either here or at the end of the sls file. The rest of the file is in `{%- if supported %} ... {%- endif %}` block.

Reply:
Ok, well, I will commit this suggestion then.